src/types/events/message-events.ts

```diff
@@ -111,6 +113,7 @@ export interface MessageRepliedEvent {
 export interface ThreadBroadcastMessageEvent {
   type: 'message';
   subtype: undefined;
```

Comment:
```suggestion
  subtype: "thread_broadcast";
```

Reply:
@aoberoi Kaz's suggestion seems like it is the right change here, but is there a particular reason `undefined` was used here as the subtype that we're not aware of?

Reply:
Nope, must have just been a typo or misunderstanding. I agree with @seratch's suggestion.